docs/data/charts/axis/GroupedYAxes.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+
+export default function GroupedYAxes() {
```

Comment:
The tick labels don't seem to be visible in dark mode:

<img width="873" height="643" alt="Image" src="https://github.com/user-attachments/assets/d0332979-5d1d-4b7e-9659-84f7aaf37c6b" />